provider/url-fetcher/index.ts

```diff
@@ -18,7 +18,8 @@ const urlFetcher: Provider<UrlFetcherSettings> = {
         return {
             // empty since we don't provide any annotations.
             selector: [],
-            meta: { name: 'URLs' },
+            name: 'URLs',
```

Comment:
```suggestion
            name: 'Web URLs',
```

to make it clearer it's about web pages